app/uk/gov/hmrc/transitmovementsrouter/models/ObjectStoreRetentionPeriod.scala

```diff
@@ -18,4 +18,4 @@ package uk.gov.hmrc.transitmovementsrouter.models
 
 import uk.gov.hmrc.objectstore.client.RetentionPeriod
 
-case class ObjectStoreRetentionPeriod(value: RetentionPeriod) extends AnyVal
+case class ObjectStoreRetentionPeriod(value: RetentionPeriod)
```

Comment:
I meant just get rid of this case class -- `RetentionPeriod` is specific enough as it is that it doesn't need wrapping.